docs/design/architectural-overview-v1.md

```diff
@@ -0,0 +1,105 @@
+# Kuadrant Architectural Overview
+
+
+<!--- variables for repeated links --->
+[AuthPolicy]: https://docs.kuadrant.io/kuadrant-operator/doc/auth/
+[RateLimitPolicy]: https://docs.kuadrant.io/kuadrant-operator/doc/rate-limiting/
+[TLSPolicy]: https://docs.kuadrant.io/kuadrant-operator/doc/tls/
+[DNSPolicy]: https://docs.kuadrant.io/kuadrant-operator/doc/reference/dnspolicy/
+
+
+## Overview
+
+Kuadrant provides connectivity, security and service protection capabilities in both a single and multi-cluster environment. It exposes these capabilities in the form of Kubernetes CRDs that implement the [Gateway API](https://gateway-api.sigs.k8s.io) concept of [policy attachment](https://gateway-api.sigs.k8s.io/reference/policy-attachment/). These policy APIs can target specific Gateway API resources such as `Gateways` and `HTTPRoutes` to extend their capabilities and configuration. They enable platform engineers to secure, protect and connect their infrastructure and allow application developers to self service and refine policies to their specific needs in order to protect exposed endpoints.  
+
+
+## Key Architectural Areas
+
+* Kuadrant architecture is defined and implemented with both control plane and data plane components.
+* The control plane is where policies are exposed and expressed as Kubernetes APIs and reconciled by a policy controller. 
+* The data plane is where Kuadrant's "policy enforcement" components exist. These components are configured by the  control plane and integrate either directly with the Gateway provider or via external integrations.
+
+## 10000m Architecture
+
+![](./images/highest-level.jpg)
+
+
+### Control Plane Components and Responsibilities
+
+The control plane is a set of controllers and operators that are responsible for for installation and configuration of other components such as the data plane enforcement components and configuration of the Gateway to enable the data plane components to interact with incoming requests. The control plane also owns and reconciles the policy CRD APIs into more complex and specific configuration objects that the policy enforcement components consume in order to know the rules to apply to incoming requests or the configuration to apply to external integrations such as DNS and ACME providers. 
+
+![](./images/control-plane-overview.jpg)
```

Comment:
What is "Limits CRD"?

Reply:
ah I see this is actually in the limitador CRD